packages/react/src/views/Message/Message.styles.js

```diff
@@ -75,7 +75,7 @@ export const getMessageDividerStyles = (theme) => {
       line-height: 1rem;
       position: relative;
       display: flex;
-      z-index: 1000;
+      z-index: 1;
```

Comment:
I'm a little afraid about z-index changes. It might disturb the existing flow in some situation. Is this required?

Reply:
![image](https://github.com/user-attachments/assets/e0105a2e-94c1-47f2-bd37-fccbd8368448)

@Spiral-Memory , actually due to a high value of z-index, the message divider was appearing on top of the sidebar like the image shown , so I had to decrease the value.